iac-extensions/kubernetes/src/test/resources/checks/MemoryLimitCheck/memory_limit_pod_with_global_limit.yaml

```diff
@@ -0,0 +1,46 @@
+# This file duplicates `memory_limit_pod.yaml` but adds a namespace to the pod.
+apiVersion: v1
+kind: Pod
+metadata:
+  name: example
+  namespace: with-global-limit
+spec:
+  containers:
+    - name: noncompliant_without_memory_limit
```

Comment:
Actually the names here and in following lines should be adjusted as they are now compliant